api/IDBTransaction.json

```diff
@@ -663,10 +663,10 @@
               "version_added": "35"
             },
             "safari": {
-              "version_added": true
+              "version_added": "10.1"
```

Comment:
Could this be a case of a false negative due to missing interface object? What does the test do in Safari 9/10?

Reply:
I looked at the commit history and it agrees with the collector's results: https://github.com/WebKit/WebKit/commit/e2bfdd0c9f5996ade274100781279db0ed00da5f

Reply:
Thanks for checking! Indeed `objectStoreNames` was added later than the interface itself.